### PR TITLE
Add a new drone voiceset for the pods.

### PR DESCRIPTION
**Pull request by GraionDilach**
Technically this is a two-years old set which I recorded for a drone in AS-as-a-mod, but since that AS is dead, I thought this might fit OpenHV.

### CODE REVIEW

**Comment by Mailaender** [comment]
That sounds pretty alien. I actually thought of scout pods as one man vehicles.

**Comment by GraionDilach** [comment]
I thought they're supposed to be robots. Okay, nevermind then.

File: mods/hv/audio/voices.yaml [hunk 1]
```diff
@@ -25,3 +25,9 @@ GenericVoice3:
 		Select: awaiting_orders, im_ready, receiving, yes_sir
 		Move: acknowledged, affirmative, as_you_wish, at_once, en route, lets_go, move_out, on_my_way, orders received, ten_four
 		Action: attack, destroy_them, hit_them_hard, light_em_up, target_locked, you_got_it
+
+PodVoice:
```
Comment (Mailaender) [inline mods/hv/audio/voices.yaml:29]: Let's use them for the drone ship instead.

**Comment by Dzierzan** [comment]
Sounds like an angry robot dog.

File: mods/hv/audio/voices.yaml (hunk 1 above)
Comment (GraionDilach) [reply mods/hv/audio/voices.yaml:29]: Alright, I've already rebased this branch locally, just let me find the actor.

**Pull request merged**